src/shell/shell_cmds.cpp

```diff
@@ -132,35 +132,48 @@ bool lookup_shell_cmd(std::string name, SHELL_Cmd &shell_cmd)
 	return true;
 }
 
-bool DOS_Shell::ExecuteConfigChange(const char* const cmd_in, const std::string args)
+bool DOS_Shell::ExecuteConfigChange(const char* const property_name, const std::string _args)
 {
 	assert(control);
 	const auto section_dosbox = static_cast<Section_prop*>(
 	        control->GetSection("dosbox"));
 
+	auto args = _args;
+	trim(args);
```

Comment:
Or since you're taking `std::string` already you can avoid this 2nd copy and just mutate it in place.

Reply:
Yeah I'd rather not mutate input args. That should be only done as a special optimisation step for performance sensitive code.